contracts/ControllerV1.sol

```diff
@@ -266,10 +289,16 @@ contract ControllerV1 is IControllerV1, SafeTeller, Ownable {
                 safeToPodId[_safeAddress] == 0,
             "Pod already exists"
         );
-        podAdmin[_podId] = _podAdmin;
+        // if pod admin, set state and lock modules
```

Comment:
I think this comment should be "if there is a pod admin", because this reads like "if the sender is pod admin"